cmd/lncli/commands.go

```diff
@@ -3365,7 +3365,8 @@ var updateChannelPolicyCommand = cli.Command{
 	Category: "Channels",
 	Usage: "Update the channel policy for all channels, or a single " +
 		"channel.",
-	ArgsUsage: "base_fee_msat fee_rate time_lock_delta [channel_point]",
+	ArgsUsage: "base_fee_msat fee_rate time_lock_delta " +
+		"[--max_htlc_msat=N] [channel_point]",
```

Comment:
Do we specify those flags like this in `ArgsUsage` for other commands?

Reply:
Got it from [this line](https://github.com/lightningnetwork/lnd/blob/master/cmd/lncli/commands.go#L152)